src/main/java/uk/gov/companieshouse/registeredemailaddressapi/eligibility/EligibilityStatusCode.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.companieshouse.registeredemailaddressapi.eligibility;
+
+public enum EligibilityStatusCode {
+    INVALID_COMPANY_WITH_NON_EXISTENT_EMAIL,
```

Comment:
Minor change. Could this be changed to INVALID_NO_REGISTERED_EMAIL_ADDRESS_EXISTS

Reply:
done